scripts/toChecksumAddress.js

```diff
@@ -0,0 +1,23 @@
+// script to convert address to checksum EIP-55
```

Comment:
Where is this script being used?

Reply:
https://github.com/maticnetwork/contracts/blob/76ac0da0861414fb8b101031759d91ebc9fd7d2f/contracts/child/MaticChildERC20.sol#L35

Reply:
oops not anymore but can we keep it in scripts as a helper method.

Reply:
thanks 😄